rl_runner.py

```diff
@@ -50,8 +50,9 @@ def loop(working_dir='estimator_working_dir', tpu_name=None):
 
         with timer("validate"):
             subprocess.call(['python', 'rl_loop.py', 'validate-hourly'] + flags)
-            subprocess.call(['python', 'main.py', 'validate',
-                             '--validate-name=pro'] + flags[3:])
+            subprocess.call(['python', 'validate.py',
+                             'gs://jacksona-sandbox/data/validate',
```

Comment:
I'm not big on this being checked in. can we move these to minigo-pub or make it a parameter

Reply:
@amj I take it this is being actively used for our validation?

Reply:
yes.